test/ErrorBoundary.test.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component }  from 'react';
+import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer'
+import ErrorBoundary from "../src/error/ErrorBoundary";
+
+class ComponentWithError extends Component {
+    render() {
+        throw new Error('No Movies Found!');
+
+      return (
+        <div>
+          <input type = "text" value = {null}/>  
+        </div>
+      );
+    }
+}
+
+describe('<ErrorBoundary> window',()=> {
+    it('should match the snapshot', () => {
+      const tree = renderer.create(<ErrorBoundary>Test</ErrorBoundary> ).toJSON()
```

Comment:
There are different methods used to render component (`renderer.create` and `shallow/mount`).
I'd suggest sticking to one, if possible (to be consistent).